src/hotspot/share/gc/g1/g1CardSet.hpp

```diff
@@ -161,6 +161,8 @@ class G1CardSetCoarsenStats {
 
   void reset();
 
+  void set_from(G1CardSetCoarsenStats& other);
```

Comment:
```suggestion
  void set(G1CardSetCoarsenStats& other);
```
Just `set()` is fine here; the reason why the other operation is called `subtract_from()` is that it subtracts the current value from the parameter, not the other way around.

Reply:
Got it, updated.